webserver/views/api/v1/core.py

```diff
@@ -126,33 +127,41 @@ def submit_low_level_nombid():
     except ValueError as e:
         raise webserver.views.api.exceptions.APIBadRequest("Cannot parse JSON document: %s" % e)
     
-    if 'musicbrainz_recordingid' in data['metadata']['tags'].keys():
-        mbid = data['metadata']['tags']['musicbrainz_recordingid']
-        action = "mbid_submitted"
-    elif 'musicbrainz_trackid' in data['metadata']['tags'].keys():
-        mbid = data['metadata']['tags']['musicbrainz_trackid']
-        action = "mbid_submitted"
+    if 'musicbrainz_recordingid' in data['metadata']['tags'].keys() or 'musicbrainz_trackid' in data['metadata']['tags'].keys():
+        raise webserver.views.api.exceptions.APIBadRequest('### The data files contains an ID this endopoint only accept nonmbid submissions!')
     elif 'md5_encoded' in data['metadata']['audio_properties'].keys():
         md5encoded = data['metadata']['audio_properties']['md5_encoded']
         mbid = db.data.find_md5_duplicates(md5encoded)
         if mbid is not None:
             action = "md5_duplicate"
-            return jsonify({"status": "OK", "itemuuid": mbid, "action": action})
+            outputmsg = {"status": "OK", "itemuuid": mbid, "action": action}
+            return jsonify(outputmsg), 200
         else:
-            mbid = str(uuid.uuid4())
-            data['metadata']['tags']['musicbrainz_trackid'] = [mbid]
-            action = "uuidV4_generate"
+            if 'artist' in data['metadata']['tags'].keys() and 'title' in data['metadata']['tags'].keys():
+                artist = data['metadata']['tags']['artist']
+                if isinstance(artist, list):
+                    artist = artist[0]
+                title = data['metadata']['tags']['title']
+                if isinstance(title, list):
+                    title = title[0]
+                artist_data = [{'artist': artist, 'title': title}]
+                mbid = messybrainz.get_messybrainz_id(artist_data)
+                if mbid is not None:
+                    action = "messybrainz_id"
+                    try:
+                        data['metadata']['tags']['musicbrainz_trackid'] = [mbid]
```

Comment:
If we add the new tag the id_type that we pass to the function in [PR195](https://github.com/metabrainz/acousticbrainz-server/pull/195/files#diff-973d6aba9f5f7b7acda5e7780e0c45a7R86) does not make much sense and we can define the id_type from the tag in which the value is stored!